web/models.py

```
@@ -2790,3 +2790,106 @@ class ScheduledPost(models.Model):
+class StudyReminder(models.Model):
+    """Model for storing study reminders and notifications."""
+    study_session = models.ForeignKey(StudySession, on_delete=models.CASCADE, related_name="reminders")
+    reminder_time = models.DateTimeField()
+    sent = models.BooleanField(default=False)
+    sent_at = models.DateTimeField(null=True, blank=True)
+    reminder_type = models.CharField(
+        max_length=20,
+        choices=[
+            ("email", "Email"),
+            ("push", "Push Notification"),
+            ("both", "Both")
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)